lms/templates/ccx/schedule.underscore

```diff
@@ -13,7 +13,7 @@
       <% _.each(chapters, function(chapter) { %>
         <tr class="chapter collapsed" data-location="<%= chapter.location %>" data-depth="1">
           <td class="unit">
-            <a href="#"><i class="fa fa-caret-right toggle-collapse"></i></a>
+            <a href="#" class="toggle-collapse"><i class="fa fa-caret-right"></i></a>
```

Comment:
More importantly, they have no text value.  At the very least you should have screen reader text that describes the control "Toggle chapter display" with an aria-expanded attribute that describes the current state (true or false)